sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -121,6 +121,11 @@ public enum AzureStackProfile
     {
         Profile20200901Hybrid = 0,
     }
+    public partial class BicepModelReaderWriterOptions : System.ClientModel.Primitives.ModelReaderWriterOptions
+    {
+        public BicepModelReaderWriterOptions() : base (default(string)) { }
+        public System.Collections.Generic.IDictionary<object, System.Collections.Generic.IDictionary<string, string>> ParameterOverrides { get { throw null; } }
```

Comment:
Should we use https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.conditionalweaktable-2?view=net-8.0 instead of `IDictionary`?

Reply:
We could, though we may also want to explore a more approachable API for adding entries which could use a ConditionalWeakTable behind the scenes.